RideShare/src/app/login/login.component.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
```

Comment:
entire file has commented out code in tests, remove and make sure they work with updated project